src/main/java/com/breskul/hw/controllers/NasaPictureController.java

```diff
@@ -0,0 +1,57 @@
+package com.breskul.hw.controllers;
+
+import com.breskul.hw.endpoints.dto.LargestNasaPictureRequest;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+@Component
+public class NasaPictureController {
+
+    Logger logger = LoggerFactory.getLogger(NasaPictureController.class);
+
+    @Autowired
+    RabbitTemplate rabbitTemplate;
+
+    @Autowired
+    NasaController nasaController;
+    Map<String, byte[]> biggestPictures = new HashMap<>();
```

Comment:
If you leave all these fields with default modifiers, it means that anyone can change the values of these fields from another calss in the current package, and as a result, you can't be sure that your classes will work properly and it will be easy to accidentally break something. Please do not ignore encapsulation and access modifiers for objects responsible for internal businnes logic.